ConsoleProject/src/services/TextureConverter.cs

```diff
@@ -182,10 +182,10 @@ public class TextureConverter {
             file.InsertRange(4, BitConverter.GetBytes(1).ToList());
 
             file.RemoveRange(8, 2);
-            file.InsertRange(8, BitConverter.GetBytes(header.FileCount).ToList());
+            file.InsertRange(8, BitConverter.GetBytes((short)header.FileCount).ToList());
 
             file.RemoveRange(10, 2);
-            file.InsertRange(10, BitConverter.GetBytes(header.FileCountB).ToList());
+            file.InsertRange(10, BitConverter.GetBytes((short)header.FileCountB).ToList());
```

Comment:
FileCount and FileCountB must be written as short, otherwise the whole file will be 4 bytes longer than expected.

Reply:
See my comment above.